darwin/torch/dataset.py

```diff
@@ -328,6 +328,11 @@ def get_target(self, index: int) -> Dict[str, Any]:
         for annotation in target["annotations"]:
             annotation_type: str = annotation.annotation_class.annotation_type
             path_key = "paths" if annotation_type == "complex_polygon" else "path"
+
+            # Darwin V2 only has paths (TODO it might be more robust fixes)
```

Comment:
is this still TODO?

Reply:
I suppose it might be more robust ways of doign this, but happy to keep it this way too.